src/python-common/ceph/deployment/service_spec.py

```diff
@@ -871,6 +874,9 @@ def __init__(self,
                  subcluster: Optional[str] = None,  # legacy, only for from_json on upgrade
                  extra_container_args: Optional[List[str]] = None,
                  custom_configs: Optional[List[CustomConfig]] = None,
+                 rgw_realm_token: Optional[str] = None,
+                 update_endpoints: Optional[bool] = False,
+                 endpoints: Optional[str] = None  # commad separated endpoints list
```

Comment:
could we call this `zone_endpoints` just so that this list of endpoints is not confused with others.

Reply:
I renamed the variable